experimental/package_crawler_static.py

```diff
@@ -0,0 +1,139 @@
+import ast
```

Comment:
Is this ready for review? If it is, can we move this to the `compatibility_lib` directory instead of in `experimental`?

Reply:
done